packages/frontend/web/src/pages/AddCocktail.tsx

```diff
@@ -0,0 +1,278 @@
+        message: 'must be a number between 1 and 3',
+      });
+    }
+  };
+
+  const squares = [
+    {
+      color: 'purple',
+      order: {
+        lg: 1,
+        md: 1,
+      },
+      biasSide: {
+        lg: ['right'],
+        md: ['right'],
+      },
+      width: {
+        lg: 110,
+        md: 105,
+      },
+      component: (
+        <LevelPart level={level} handleClick={handleClick} errors={errors} />
+      ),
+    },
+    {
+      color: 'yellow',
+      order: {
+        lg: 3,
+        md: 4,
+      },
+      biasSide: {
+        lg: ['right', 'left'],
+        md: ['left'],
+      },
+      width: {
+        lg: 130,
+        md: 116,
+      },
+      right: {
+        lg: 15,
+        md: 16,
+      },
+      component: (
+        <AlcoholPart
+          handleClickAlcohol={handleClickAlcohol}
+          watch={watch}
+          errors={errors}
+        />
+      ),
+    },
+    {
+      color: 'blue',
+      order: {
+        lg: 5,
+        md: 2,
+      },
+      biasSide: {
+        lg: ['left'],
+        md: ['left'],
+      },
+      width: {
+        lg: 108,
+        md: 105,
+      },
+      right: {
+        lg: 8,
+        md: 0,
+      },
+      component: (
+        <IngredientsPart
+          register={register}
+          selectedIngredient={selectedIngredient}
+          handleIngredientChange={handleIngredientChange}
+          errors={errors}
+        />
+      ),
+    },
+    {
+      color: 'orange',
+      order: {
+        lg: 2,
+        md: 5,
+      },
+      biasSide: {
+        lg: ['right'],
+        md: ['right'],
+      },
+      width: {
+        lg: 110,
+        md: 110,
+      },
+      right: {
+        lg: 0,
+        md: 10,
+      },
+      component: <GlassPart register={register} errors={errors} />,
+    },
+    {
+      color: 'green',
+      order: {
+        lg: 4,
+        md: 3,
+      },
+      biasSide: {
+        lg: ['right', 'left'],
+        md: ['right'],
+      },
+      width: {
+        lg: 103,
+        md: 105,
+      },
+      component: (
+        <ToppingPart
+          register={register}
+          selectedTopping={selectedTopping}
+          handleToppingChange={handleToppingChange}
+          errors={errors}
+        />
+      ),
+    },
+    {
+      color: 'pink',
+      order: {
+        lg: '6',
+        md: '6',
+      },
+      biasSide: {
+        lg: ['left'],
+        md: ['left'],
+      },
+      width: {
+        lg: 104,
+        md: 105,
+      },
+      right: {
+        lg: 4,
+        md: 5,
+      },
+      component: (
+        <NamePart
+          register={register}
+          handleErrorSubmit={handleErrorSubmit}
+          errors={errors}
+        />
+      ),
+    },
+  ];
+
+  return (
+    <form className='flex justify-center' onSubmit={handleSubmit(onSubmit)}>
+      <div className='grid w-screen grid-flow-col grid-rows-6 gap-1 gap-y-2 md:h-screen md:grid-rows-3 md:p-3 lg:grid-rows-2'>
+        {squares.map((div, index) => (
```

Comment:
```suggestion
        {squares.map((div, index) => (
```
Pourquoi `div` ? Ca porte à confusion avec la balise `div`